#!/usr/bin/env python
PACKAGE = "stefmap_ros"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("start_predictions" ,  bool_t,   0, "Start the model predictions", False)
gen.add("time_16_09_2019",    int_t,    0, "Hour of the day", 12,  0, 24)
gen.add("model_order",        int_t,    0, "Number of frequency components used", 0, 0, 10)


exit(gen.generate(PACKAGE, "stefmap_ros", "stefmapclient"))